en/docs/design-your-project/index.md

```diff
@@ -90,50 +90,6 @@ To set up a Ballerina project as a VS Code workspace, add all the Ballerina pack
     !!! Info 
         The [workspace file](https://github.com/ballerina-guides/gcp-microservices-demo/blob/main/project.code-workspace) of the Online Boutique microservices demo is available in the repository. 
 
-Once the packages are added to the workspace, activate the Architecture View via the `Ballerina: Architecture View` command in the command palette.
+3. Once the packages are added to the workspace, activate the Architecture View via the `Ballerina: Architecture View` command in the command palette.
```

Comment:
Is it correct to use number `3` for this? The first and second points are part of `add all the Ballerina packages to the workspace using one of the following methods`. So this number `3` is not another method to add packages to workspace.

Reply:
Removed the number 3 and made it a paragraph.